Runner.java

```diff
@@ -0,0 +1,66 @@
+import beans.ExtraTrial;
+import beans.LightTrial;
+import beans.StrongTrial;
+import beans.Trial;
+
+import java.util.*;
+
+public class Runner {
+    public static void main(String[] args) {
+        List<Trial> trials = new ArrayList<>(List.of(
+                new Trial("Trial1", 35, 60),
+                new Trial("Trial2", 53, 46),
+                new Trial("Trial3", 27, 16),
+                new LightTrial("LightTrial1", 84, 95),
+                new LightTrial("LightTrial2", 76, 59),
+                new StrongTrial("StrongTrial1", 83, 78),
+                new StrongTrial("StrongTrial2", 64, 73),
+                new ExtraTrial("ExtraTrial1", 53, 75, 69),
+                new ExtraTrial("ExtraTrial2", 42, 36, 75)));
+        List<Trial> failedTrials = new ArrayList<>();
+
+        for (Trial trial : trials) {
+            System.out.println(trial);
+        }
```

Comment:
no loop is needed here and further. 
use stream intermediate and terminal operations. 
see [Aggregate Operations](https://docs.oracle.com/javase/tutorial/collections/streams/index.html) and my guide.